### PR TITLE
PR to merge main into production

### PR DESCRIPTION
**Pull request by mwazir**
Creating pull request to merge main into production

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,219 @@
...
+              updateRestaurantName={(event) => handleUpdateRestaurant(event)}
+              grossBillAmountState={grossBillAmount}
+              updateGrossBillAmount={(event) => handleUpdateGrossBillAmount(event)}
+              grossTipAmountState={grossTipAmount}
+              updateGrossTipAmount={(event) => handleUpdateGrossTipAmount(event)}
+              groupSizeState={groupSizeInput}
+              updateGroupSize={(event) => handleUpdateGroupSize(event)}
+              customTipEnabledState={customTipEnabled}
+              tipTextInputValueState={tipTextInputValue}
+              // checkedStatus={}
```
Comment (tdmitchell) [inline src/App.js]: You can remove this code

File: src/components/BillDisplay.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const BillDisplay = (props) => {
+    return (
+        <div className="bill-display-container">
+            <div className="gross-bill-per-person">
+                <div className="display-label-container">
+                        <p>Gross Bill</p>
```
Comment (tdmitchell) [inline src/components/BillDisplay.js:6]: Take a look at the indentation for lines 6 and 7.

File: src/components/BillDisplay.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                <div className="display-label-container">
+                    <p>Tip Amount</p>
+                    <p className="per-person-label">/ Person</p>
+                </div>
+                <p className="amount-display">{props.tipPerPersonState}</p>
+            </div>
+            <div className="total-bill-per-person">
+                <div className="display-label-container">
+                    <p>Total Bill</p>
+                        <p className="per-person-label">/ Person</p>
```
Comment (tdmitchell) [inline src/components/BillDisplay.js]: Take a look at the indentation for line 22

File: src/styles/sass/pages/_home.scss [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+            text-align: center;
+            color: $clr-grey;
+        }
+
+        @media screen and (max-width: 480px) {
+            width: 100%;
+        }
+    }
+
+    @media screen and (max-width: 480px) {
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss]: I miss the media query for some intermediary devices. Maybe you could implement it for the next split.

File: src/styles/sass/base/_base.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    background: $clr-lightGreen;
+}
+
+/* Removing default number input styles (increase/decrease arrows)
```
Comment (tdmitchell) [inline src/styles/sass/base/_base.scss:11]: I think you can remove this line.

File: src/styles/sass/pages/_home.scss [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+    justify-content: center;
+    align-items: center;
+}
+
+
+/* Bill Splitter App Styles */
+header {
+    width: 100%;
+}
+
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss:14]: You can remove this line

File: src/styles/sass/pages/_home.scss [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+
+h1 {
+    width: 100%;
+    margin: 3rem 0 3rem 0;
+    text-align: center;
+    color: $clr-grey;
+}
+
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss:21]: You can remove this line

File: src/styles/sass/pages/_home.scss [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+.bill-form-container {
+    width: 50%;
+    border-radius: 25px;
+    background: $clr-white;
+    padding: 2rem;
+
+    @media screen and (max-width: 1200px) {
+        width: 100%;
+    }
+
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss:44]: You can remove this line

File: src/styles/sass/pages/_home.scss [hunk 9]
```diff
@@ -0,0 +1,316 @@
...
+
+    form {
+        display: flex;
+        flex-direction: column;
+    }
+
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss:50]: You can remove this line

File: src/styles/sass/pages/_home.scss [hunk 10]
```diff
@@ -0,0 +1,316 @@
...
+        thead tr {
+            position: absolute;
+            top: -9999px;
+            left: -9999px;
+        }
+
+        td {
+            position: relative;
+            padding-left: 50%;
+
```
Comment (tdmitchell) [inline src/styles/sass/pages/_home.scss]: You can remove this line

**Review by tdmitchell — APPROVED** [review]
Great work, there are 2 files that you don't have any code in it, and some small cleaning up to do.
You can implement the responsiveness for other devices later on.
Files:
src/styles/sass/layout/_navigation.scss
src/styles/sass/pages/_history.scss

File: src/App.js (hunk 1 above)
Comment (mwazir) [reply src/App.js]: Done

File: src/components/BillDisplay.js (hunk 2 above)
Comment (mwazir) [reply src/components/BillDisplay.js:6]: Done

File: src/components/BillDisplay.js (hunk 3 above)
Comment (mwazir) [reply src/components/BillDisplay.js]: Done

File: src/styles/sass/base/_base.scss (hunk 5 above)
Comment (mwazir) [reply src/styles/sass/base/_base.scss:11]: Done

File: src/styles/sass/pages/_home.scss (hunk 4 above)
Comment (mwazir) [reply src/styles/sass/pages/_home.scss]: Sounds good, noted.

File: src/styles/sass/pages/_home.scss (hunk 6 above)
Comment (mwazir) [reply src/styles/sass/pages/_home.scss:14]: Done

File: src/styles/sass/pages/_home.scss (hunk 7 above)
Comment (mwazir) [reply src/styles/sass/pages/_home.scss:21]: Done

File: src/styles/sass/pages/_home.scss (hunk 8 above)
Comment (mwazir) [reply src/styles/sass/pages/_home.scss:44]: Done

File: src/styles/sass/pages/_home.scss (hunk 10 above)
Comment (mwazir) [reply src/styles/sass/pages/_home.scss]: Done

**Pull request merged**